### PR TITLE
Update App.tsx

### PR DESCRIPTION
**Pull request by SPLATPLAYS**
fixed errors

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,17 +1,32 @@
+import { useState } from "react";
```
Comment (spacey-sooty) [inline src/App.tsx]: React is not a dependency of this project its a preact project

**Review by spacey-sooty — CHANGES_REQUESTED** [review]

**Comment by spacey-sooty** [comment]
This doesn't fix any of the errors btw they are from `e.target.value`

**Comment by SPLATPLAYS** [comment]
Made some updates, does this work?

File: src/App.tsx [hunk 2]
```diff
@@ -1,17 +1,40 @@
+import { h } from 'preact';
```
Comment (spacey-sooty) [inline src/App.tsx]:
```suggestion
```

File: src/App.tsx [hunk 3]
```diff
@@ -1,17 +1,40 @@
...
 
 function App() {
-    var teamNum = 9999;
+    const [teamNum, setTeamNum] = useState(9999);
+    const [mode, setMode] = useState("teleop");
+
+    const handleTeamNumChange = (e) => {
+        const newValue = e.target.value;
+        setTeamNum(Number(newValue));
+    };
```
Comment (spacey-sooty) [inline src/App.tsx:12]: This approach doesn't fix the issue...

**Review by spacey-sooty — CHANGES_REQUESTED** [review]

**Pull request closed without merging**